parameters1.json

```diff
@@ -0,0 +1,14 @@
+{
+    "testconnection": {
+        "SNOWFLAKE_TEST_USER":      "admin",
+        "SNOWFLAKE_TEST_PASSWORD":  "test",
+        "SNOWFLAKE_TEST_ACCOUNT":   "snowflake",
+        "SNOWFLAKE_TEST_WAREHOUSE": "xsmall",
+        "SNOWFLAKE_TEST_DATABASE":  "SNOWTRAIL_TEST",
+        "SNOWFLAKE_TEST_SCHEMA":    "public",
+        "SNOWFLAKE_TEST_ROLE":      "ACCOUNTADMIN",
+        "SNOWFLAKE_TEST_PROTOCOL":  "http",
+        "SNOWFLAKE_TEST_HOST":      "snowflake.qa1.int.snowflakecomputing.com",
+        "SNOWFLAKE_TEST_PORT":      "8084"
+    }
+}
```

Comment:
please don't include this.

Reply:
.github/workflows/parameters* already have the encrypted credentials, so you can just use them